it/uk/gov/hmrc/agentassurance/controllers/AgentAssuranceControllerISpec.scala

```diff
@@ -439,6 +439,21 @@ class AgentAssuranceControllerISpec extends IntegrationSpec
       Then("404 NOT_FOUND is returned")
       updateResponse.status shouldBe 404
     }
+
+//    scenario("ARN to be unique to each UTR in the MDTP database") {
+//
+//      await(repo.insert(AmlsEntity(utr, amlsDetails, None, LocalDate.now())))
+//      await(repo.insert(AmlsEntity(Utr("8588532862"), amlsDetails, Some(arn), LocalDate.now())))
+//
+//      Given("User is logged in and is an agent")
+//      withAffinityGroupAgent
+//
+//      When("PUT /amls/utr/:identifier is called")
+//      val updateResponse: WSResponse = doUpdate
+//
+//      Then("400 BAD_REQUEST is returned")
+//      updateResponse.status shouldBe 400
```

Comment:
for some reason this test is not working it returns 200 here even for duplicate ARN which means the uniqueness of index is not working - 
Its working fine when i tested via curl/browser
any ideas ?